src/Schema/Builder.php

```diff
@@ -165,9 +178,42 @@ public function getTables($schema = null)
             ];
         }
 
-        usort($collections, function ($a, $b) {
-            return $a['name'] <=> $b['name'];
-        });
+        usort($collections, fn ($a, $b) => $a['name'] <=> $b['name']);
+
+        return $collections;
+    }
+
+    /** @param  string|null $schema Database name */
+    public function getViews($schema = null)
+    {
+        $db = $this->connection->getDatabase($schema);
+        $collections = [];
+
+        foreach ($db->listCollections() as $collectionInfo) {
+            $collectionName = $collectionInfo->getName();
+
+            // Skip system collections
+            if (str_starts_with($collectionName, 'system.')) {
+                continue;
+            }
+
+            // Skip normal type collection
+            if ($collectionInfo->getType() !== 'view') {
+                continue;
+            }
+
+            $collections[] = [
+                'name' => $collectionName,
+                'schema' => $db->getDatabaseName(),
+                'schema_qualified_name' => $db->getDatabaseName() . '.' . $collectionName,
+                'size' => null,
```

Comment:
If this is relevant, the information can be provided via [`$collStats`](https://www.mongodb.com/docs/manual/reference/operator/aggregation/collStats/) (MongoDB 6.2+). I'm not sure that'd be worth the overhead to obtain this for all collections/views, though.

Reply:
If someone finds it useful, we can do it, but we'll have to check the impact on performance if there are a lot of collections.